### ISSUE TITLE
User guide example error

### ISSUE THREAD

**Issue by ahuang11** [ioam/parambokeh#63]
From user guide example: 
https://ioam.github.io/parambokeh/user_guide/View_Parameters.html

holoviews=1.10.5
parambokeh=0.2.2

```
/mnt/c/Users/sephi/GOOGLE~1/Bash/parambokeh/parambokeh/view.py in render_function(obj, view)
     16         plot = renderer.get_plot(obj, doc=view._document)
     17         if view._notebook:
---> 18             from holoviews.plotting.comms import JupyterComm
     19             comm = JupyterComm(plot, view._comm_target)
     20             plot.comm = comm


ModuleNotFoundError: No module named 'holoviews.plotting.comms'
```

I tried replacing that line with `from pyviz_comms import JupyterComm` but then I get a new error stemming from `on_init=True` input

```
     32 example = CurveExample(name='HoloViews Example')
     33 parambokeh.Widgets(example, callback=example.event, push=False,
---> 34                    on_init=True, view_position='right')

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in __new__(class_, *args, **params)
   1889         inst = class_.instance()
   1890         inst._set_name(class_.__name__)
-> 1891         return inst.__call__(*args,**params)
   1892 
   1893     def __call__(self,*args,**kw):

/mnt/c/Users/sephi/GOOGLE~1/Bash/parambokeh/parambokeh/__init__.py in __call__(self, parameterized, doc, plots, **params)
    187 
    188         if self.p.on_init:
--> 189             self.execute()
    190 
    191         if self.p.mode == 'raw':

/mnt/c/Users/sephi/GOOGLE~1/Bash/parambokeh/parambokeh/__init__.py in execute(self, changed)
    381         if self.p.callback is not None:
    382             if get_method_owner(self.p.callback) is self.parameterized:
--> 383                 self.p.callback(**changed)
    384             else:
    385                 self.p.callback(self.parameterized, **changed)

<ipython-input-8-65291cacb15b> in event(self, **kwargs)
     26     def event(self, **kwargs):
     27         if not self.output or any(k in kwargs for k in ['color', 'element']):
---> 28             self.output = hv.DynamicMap(self.view, streams=[self])
     29         else:
     30             super(CurveExample, self).event(**kwargs)

/mnt/c/Users/sephi/GOOGLE~1/Bash/parambokeh/parambokeh/view.py in __set__(self, obj, val)
     48         obj_id = id(obj)
     49         if obj_id in self.callbacks:
---> 50             self.callbacks[obj_id](self.renderer(val, self))
     51 
     52 

/mnt/c/Users/sephi/GOOGLE~1/Bash/parambokeh/parambokeh/view.py in render_function(obj, view)
     17         if view._notebook:
     18             from pyviz_comms import JupyterComm
---> 19             comm = JupyterComm(plot, view._comm_target)
     20             plot.comm = comm
     21         plot.document = view._document

~/anaconda3/lib/python3.6/site-packages/pyviz_comms/__init__.py in __init__(self, id, on_msg)
    193         self._on_msg = on_msg
    194         self._comm = None
--> 195         super(Comm, self).__init__(id = id if id else uuid.uuid4().hex)
    196 
    197 

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in __init__(self, **params)
   1039         self.__generate_name()
   1040 
-> 1041         self._setup_params(**params)
   1042         object_count += 1
   1043 

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in override_initialization(parameterized_instance, *args, **kw)
    976         original_initialized=parameterized_instance.initialized
    977         parameterized_instance.initialized=False
--> 978         fn(parameterized_instance,*args,**kw)
    979         parameterized_instance.initialized=original_initialized
    980     return override_initialization

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in _setup_params(self, **params)
   1439                 self.warning("Setting non-parameter attribute %s=%s using a mechanism intended only for parameters",name,val)
   1440             # i.e. if not desc it's setting an attribute in __dict__, not a Parameter
-> 1441             setattr(self,name,val)
   1442 
   1443 

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in __set__(self, obj, val)
    535 
    536     def __set__(self,obj,val):
--> 537         self._check_value(val)
    538         super(String,self).__set__(obj,val)
    539 

~/anaconda3/lib/python3.6/site-packages/param/parameterized.py in _check_value(self, val)
    532     def _check_value(self,val):
    533         if not isinstance(val, self.basestring) and not (self.allow_None and val is None):
--> 534             raise ValueError("String '%s' only takes a string value."%self._attrib_name)
    535 
    536     def __set__(self,obj,val):```

**Comment by philippjfr** [comment]
We need a new parambokeh release asap, this is fixed on master.

**Comment by ceball** [comment]
This should be fixed now in 0.2.3. Please re-open if you still have problems.
